go.mod

```diff
@@ -1,4 +1,4 @@
-// +heroku goVersion go1.19
```

Comment:
Is this supposed to be `21`?

Reply:
Its intentional, since Heroku doesn't support 1.21 yet (https://github.com/heroku/heroku-buildpack-go/pull/521)